src/components/dashboard/formResponses/formResponseDetails/formResponseDetails.jsx

```diff
@@ -8,6 +8,18 @@ const FormResponseDetails = ({ showApproveModal, showRejectModal }) => {
   if (!response) return null;
 
   const { questions } = response;
+  const isApprovedOrRejected = response.statusString === 'Approved' || response.statusString === 'Rejected';
+  const noop = Function.prototype;
+
+  let approveButton;
+  let rejectButton;
+  if (isApprovedOrRejected) {
+    approveButton = <button onClick={noop} disabled>Approve</button>;
+    rejectButton = <button onClick={noop} disabled>Reject</button>;
```

Comment:
@mbareta I think noop makes it a bit more expressive, not that it's a big difference or anything. :)